capstone/pom.xml

```diff
@@ -36,6 +36,33 @@
       <artifactId>spring-cloud-gcp-starter-data-datastore</artifactId>
       <version>1.2.3.RELEASE</version>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
```

Comment:
Are you using any of the AppEngine APIs? Since we're not using the Users API, I don't think so.
Spring Cloud GCP should have everything you'll need.